libs/domain/picking/offline/services/sw-push-initializer.service.ts

```diff
@@ -13,6 +14,9 @@ export class SwPushInitializerService implements AppInitializer {
   constructor(private syncSchedulerService = inject(SyncSchedulerService)) {}
 
   initialize(): void | Observable<void> | Promise<void> {
+    //make subscription on auth token
+    resolve(OauthService).invokeStoredToken();
```

Comment:
Why not `inject()` it in the constructor?

Reply:
I'm not sure that there is a big difference, due to we make resolving only once, but ok